check_tv.py

```
@@ -315,26 +374,29 @@ def checkarg():
     except:
           logging.info("failed script shutdown")
   except Exception as e:
+   try:
     logging.info("theres no arg")
     arg = "Null"
     selwebdriver_check(arg, arg, "Null")
+   except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)